### ISSUE TITLE
dnd (do not disturb) integration.

### ISSUE THREAD

**Issue by francesco-cadei** [vmiklos/plees-tracker#419]
I think would be excellent to have the possibility to activate dnd during plees-tracker monitor.

**Comment by vmiklos** [comment]
Do you know some other open source app that turns on/off the DND mode automatically? That would be a useful code pointer.

In general, I'm not against this, although I have DND turned on/off based on clock (and probably I'm not alone), so this should not be on by default.

**Comment by francesco-cadei** [comment]
I found only this [https://github.com/dic1911/android_AutoDND](https://github.com/dic1911/android_AutoDND) but on my android 13 seems not work.

I think the option is interesting because I not go to sleep always at the same time, specially on weekend.

**Comment by MachineThing** [comment]
Implemented in #448

**Comment by vmiklos** [comment]
Indeed, thanks! Will cut a next release early March for the ones who don't just build master from source.
